lib/pages/scout/dialogs/redeem_dialog.dart

```diff
@@ -407,7 +407,7 @@ class _RedeemDialogState extends State<RedeemDialog> {
                             context: context,
                             builder: (BuildContext context) =>
                                 confirmTransaction(context, true, ''),
-                          );
+                          ).then((value) => Navigator.pop(context));
```

Comment:
You should check for `mounted` before calling `pop`.

Reply:
Done